tests/helpers/Users/login.test.js

```diff
@@ -0,0 +1,27 @@
+const request = require("supertest");
+const createUser = require("../../../helpers/Users/createUser");
+const { createTables, wipeDBTables } = require("../../../db/databaseHelpers");
+const app = require("../../../app");
+
+beforeAll(async() =>{
+  await createTables();
+  await createUser("1", "txiong@", "password", "student");
+});
+afterAll(async() =>{
+  await wipeDBTables();
+});
+
+describe("login endpoint test", () =>{
```

Comment:
This is a helpers file instead of a controller.

Reply:
Login is not a helpers file